module4/auth/src/main/java/com/epam/esm/security/service/util/JwtTokenProvider.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.esm.security.service.util;
+
+import com.epam.esm.security.exception.ExceptionMessageValue;
+import com.epam.esm.security.exception.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.naming.AuthenticationException;
+import javax.servlet.http.HttpServletRequest;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Base64;
+import java.util.Date;
+import java.util.Optional;
+
+import static org.springframework.util.StringUtils.hasText;
+
+@Component
+@RequiredArgsConstructor
+public class JwtTokenProvider {
+
+    private final String PREFIX_FOR_POSTMAN_AUTH = "Bearer ";
```

Comment:
it's not postman specific, it's a token specific prefix for Authorization header

Reply:
exactly! done